administration/src/bp-modules/stores/StoresCSVInput.tsx

```diff
@@ -102,6 +113,11 @@ const StoresCsvInput = ({ setAcceptingStores, fields }: StoresCsvInputProps): Re
       }
       const acceptingStores = lines.map((line: string[]) => lineToStoreEntry(line, csvHeader, fields))
 
+      if (hasStoreDuplicates(acceptingStores)) {
+        showInputError(`Die CSV enthält doppelte Einträge.`)
```

Comment:
nice to have, but might be tricky to implement: add line numbers to the message.
because it might be difficult to find the duplicates among thousands of entries to fix the issue.